src/components/screens/PointOfInterest.js

```diff
@@ -210,11 +210,20 @@ export const PointOfInterest = ({ data, hideMap, navigation, route }) => {
                 [category.iconName || MAP.DEFAULT_PIN]: 1,
                 iconName: category.iconName || MAP.DEFAULT_PIN,
                 id,
-                position: { latitude, longitude }
+                position: { latitude, longitude },
+                serviceName: title,
+                title
               }
             ]}
-            selectedMarker={id}
             mapStyle={styles.mapStyle}
+            interactivity={{
+              pitchEnabled: false,
+              rotateEnabled: false,
+              scrollEnabled: false,
+              zoomEnabled: false
+            }}
```

Comment:
We can't scroll the map, we can't zoom in, but we can show our own position. Is that right? If I am too far away and once I press the show my location button, I can no longer go back to the location that shows the location of the POI.

<img width="411" alt="image" src="https://github.com/user-attachments/assets/6875ab02-9f98-4c45-9ae9-abf21cad29b2" />

Reply:
I added the `isMyLocationButtonVisible` prop to the component as `false` and now the button is not visible.